utils/scheduler.py

```diff
@@ -0,0 +1,29 @@
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from utils.core import get_random_quote
+import logging
+from logging.config import fileConfig
+
+fileConfig('../utils/logging_config.ini')
+logger = logging.getLogger('discord')
+
+
+def init_message_scheduler(config, client):
```

Comment:
I will leave this up to you, but this format of passing the entire config in is inconsistent with utils.core, where we only pass in the needed config, which in this case is config.get("SCHEDULES"). We can leave it as is and discuss the preferred method later, but eventually it would be preferable to be consistent. Basically: Pass in entire config to everything? Or pass in the sub-config relevant to that function?